FeedStoreChallenge/Infrustructure/ManagedCache.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Zheng Kanyan on 2021/6/11.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
```

Comment:
`ManagedCache` is not supposed to be inherited from, so it should be marked as final.
Also, the convention in this codebase is to omit the default access level of `internal`, so it can be removed.
```suggestion
final class ManagedCache: NSManagedObject {
```